publish_example_service_test.go

```diff
@@ -45,7 +44,7 @@ func exampleserviceIsRunWithSnetdaemon(table *gherkin.DataTable) error {
     }`
```

Comment:
WIRE_ENCODING, SERVICE_TYPE, POLL_SLEEP, LOG_LEVEL are not required anymore

Reply:
I will remove the example-services from the test as obsolete.

Reply:
I removed the example-service test.